packages/@okta/vuepress-site/docs/guides/protect-your-api/main/index.md

```diff
@@ -30,27 +30,39 @@ A [custom authorization server](/docs/concepts/auth-servers/#custom-authorizatio
 
 ## Set up Okta
 
-Set up your [Okta org](/docs/concepts/okta-organizations/). The CLI is the quickest way to work with your Okta org, so we recommend using it for the first few steps. If you don't want to install the CLI, you can [manually sign up for an org](https://developer.okta.com/signup/) instead. We provide non-CLI instructions along with the CLI steps below.
+Set up your developer account and [Okta org](/docs/concepts/okta-organizations/). The Okta command-line interface (CLI) is the quickest way to do this. Alternatively, you can [manually sign up for a developer account](https://developer.okta.com/signup/).
 
-1. Install the Okta command-line interface: [Okta CLI](https://cli.okta.com/).
-2. If you don't already have a free Okta developer account, create one by entering `okta register` on the command line.
-3. Make a note of the Okta Domain as you need that later.
-4. **IMPORTANT:** Set the password for your Okta developer org by opening the link that's shown after your domain is registered. Look for output similar to this:
+1. Install [Okta CLI](https://cli.okta.com/).
+1. Open your terminal.
+1. Run `okta register`, and enter your first name, last name, email address, and country.
```

Comment:
would it be easier to do something along the lines of "and enter your details/information."? The prompts from the program should be self-explanatory.

Reply:
If I change that, the next instructions referring to the email address makes less sense.